src/search/search_engines/breadth_first_search.cc

```diff
@@ -59,14 +59,13 @@ int BreadthFirstSearch<PackedStateT>::search(const Task &task,
 
         assert(sid.id() >= 0 && (unsigned) sid.id() < space.size());
 
-        auto successors = generator.generate_successors(task.actions, packer.unpack(space.get_state(sid)), task.static_info);
+        const auto &state = packer.unpack(space.get_state(sid));
```

Comment:
This should not be a reference, as we're returning an object in the unpack method!